microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -12,12 +12,16 @@ toc::[]
 
 [id="microshift-4-14-about-this-release"]
 == About this release
+// TODO: Update with the relevant information closer to release.
+{product-title} (link:https://access.redhat.com/errata/RHSA-2023:XXXX[RHSA-2023:XXXX]) is now Generally Available. This release uses link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md[Kubernetes 1.27] with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} {product-version} are included in this topic.
 
-//The Red Hat build of {product-title} is Technology Preview only. Features and known issues that pertain to {product-title} {ocp-version} are included in this topic. This Technology Preview software is not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using {product-title} in production. Technology Preview provides early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+You can deploy {product-title} clusters to either on-premises, cloud, or disconnected environments.
 
-//For more information about the support scope of Red Hat Technology Preview features, read link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+// Double check OP system versions
+{product-title} {product-version} is supported on {op-system-base-full} 9 as well as on {op-system-ostree-first} 9.2.
 
```

Comment:
Why to we point to generic RHEL 9, but specifically ostree 9.2? We should make clear it is RHEL 9.2 what is currently supported. And that both rpm and ostree based flavours of RHEL are supported.